### PR TITLE
Add a shield/background to the busy indicator message to increase readability

### PR DESCRIPTION
**Pull request by nirvn**
Fixes https://github.com/opengisch/QField/issues/5039

Looks like this:

image

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 3d392154964dd0dad4c784eb10ff03ac39103a8c: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-5126-5c20294f1fab51b699cdc3bfc96ee49870aae8c1-android-arm64.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-5126-5c20294f1fab51b699cdc3bfc96ee49870aae8c1-android-arm-neon.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-5126-5c20294f1fab51b699cdc3bfc96ee49870aae8c1-android-x64.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-5126-5c20294f1fab51b699cdc3bfc96ee49870aae8c1-android-x86.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-5126-5c20294f1fab51b699cdc3bfc96ee49870aae8c1-android-arm64.apk)
- [all access arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-5126-5c20294f1fab51b699cdc3bfc96ee49870aae8c1-android-arm-neon.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-5126-5c20294f1fab51b699cdc3bfc96ee49870aae8c1-android-x64.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-5126-5c20294f1fab51b699cdc3bfc96ee49870aae8c1-android-x86.apk)
</details>

File: src/qml/BusyOverlay.qml [hunk 1]
```diff
@@ -69,13 +69,34 @@ Rectangle {
...
+    Text {
+      id: busyMessage
+
+      property int absoluteWidth: busyMessageFontMetrics.boundingRect(text).width + 10
+      width: Math.min(mainWindow.width - 20, absoluteWidth)
+
+      anchors.centerIn: parent
+      horizontalAlignment: Text.AlignHCenter
+      font: Theme.tipFont
+      color: "white"
```
Comment (suricactus) [inline src/qml/BusyOverlay.qml:92]: I guess hardworking this as white will always work fine with  `Theme.darkGraySemiOpaque`. Any chance we can change this hard-coded color to something from theme?

**Review by suricactus — APPROVED** [review]
Nice improvement!

File: src/qml/BusyOverlay.qml (hunk 1 above)
Comment (nirvn) [reply src/qml/BusyOverlay.qml:92]: @suricactus , I'd like to defer that to a/ a different PR, and b/ whenever we'll need to change throughout the code. We use "white" for all tool buttons too, which is the main context this color is used against. 

Point taken though :)

**Pull request merged**